clang-format/blank.txt

```diff
@@ -0,0 +1 @@
+This folder is intentionally left blank.
```

Comment:
Usually, for this pattern, we usually see an empty file named `.gitkeep`, like explained in https://gist.github.com/jlollis/54d3b3a7ed12776ac9f0d4bc13e63ed9